src/graphql/getDownloadURL.graphql

```diff
@@ -1,6 +1,6 @@
-query getDownloadURL {
-  repository(owner: "friendsofagape", name: "autographa") {
-    releases(last: 10, orderBy: {field: CREATED_AT, direction: DESC}) {
+query getDownloadURL { 
+    repository(owner: "friendsofagape", name: "autographa") {
+    releases(first: 10, orderBy: {field: CREATED_AT, direction: DESC}) {
```

Comment:
Does this only return the non-prereleases because that is what we want.

Reply:
No, this returns all the releases. But, there is function that  filters out the pre-releases.